### ISSUE TITLE
Title attribute should be removed

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#99]
URL:  https://hcdev.openplus.ca/en/user/login
User Agent: Chrome
Language: both EN and FR

Issue Description: The “title” attribute of the ‘Log in’ / ‘Se connecter’ button says ‘Search’ / Recherche. It is not correct.  In this case, the “title” attribute is not necessary and should be removed.

Steps to reproduce:
1.	Navigate to the Log in
2.	View code for the Submit button or hover your mouse over the button, the tooltip will display the "title" attribute.

Expected results: Remove the title attribute it is unnecessary as it provides no additional information to users.
![image](https://user-images.githubusercontent.com/68117281/136404814-a03f10e7-d363-4939-985d-d7ce6627ab20.png)

**Comment by qanitah** [comment]
This also applies to search page https://hcdev.openplus.ca/en/search/site

The “title” attribute of the “Reset” button says “Search”. In this case, the “title” attribute is not necessary and should be removed. Also, the “title” attribute from the “Search” should be removed as well since it is the same value and doesn’t add any value to button’s description.

![image](https://user-images.githubusercontent.com/14280062/136609840-b086be44-de4a-4717-b7d6-128c2388e1bd.png)

**Comment by qanitah** [comment]
Similar issue on Recherche page https://hcdev.openplus.ca/fr/search/site

![image](https://user-images.githubusercontent.com/14280062/136610669-b4e81efe-7039-44c9-8664-52cc06a5cff7.png)

**Comment by qanitah** [comment]
Some other pages also have incorrect title attributes. 

![image](https://user-images.githubusercontent.com/14280062/136615650-ec24d104-480b-4794-8a6e-b4fe2539ec97.png)

![image](https://user-images.githubusercontent.com/14280062/136615661-6c3bece3-0ed0-4e63-a328-2eb159cadea2.png)


![image](https://user-images.githubusercontent.com/14280062/136615678-93fcaf82-da63-4a89-a35c-07521813f8b1.png)

![image](https://user-images.githubusercontent.com/14280062/136615696-5619a43a-538b-4d20-b3e7-a141849bce4e.png)

**Comment by echouin** [comment]
@qanitah added public facing label as some of these issues are on the public website.

**Comment by ndolter** [comment]
Validated that the title attribute has been removed from the following pages based on the screen shots provided above:
- Log In 
- Search 
- Edit Alert / Recall test - Oct 14: Save, Cancel, Remove & Add row buttons (go to:  Admin side, under Dashboard, click the build button associated with the alert "test - Oct 14", then click the Edit button associated with each section. The add row button is under this section Section 3 - Table input)
- Dashboard
